docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -740,20 +786,20 @@ An example of parsing a XML config file using JAXB is shown in the `TestProcesso
 <2> If found, parse using JAXB context for `XmlConfig.class`
 
 [NOTE]
-====
+=====
```

Comment:
Same here and for all the other `=====` on a single line. A simple regexp replace should do it: `sed -r -i 's/^=====$/====/g' docs/src/main/asciidoc/extension-authors-guide.adoc`.